Tutorials/whatiswot/3-JSON/3-JSON.vtt

```diff
@@ -0,0 +1,98 @@
+WEBVTT
+
+00:00:00.000 --> 00:00:06.737 
+In this video, we will take a closer look at JSON. Since it is the most used
+
+00:00:06.737 --> 00:00:13.475 
+payload format in the Web of Things , as well as the way Thing Descriptions are represented.
+
+00:00:13.475 --> 00:00:17.837 
+So, let’s start with what is JSON.
+
+00:00:17.837 --> 00:00:22.312 
+JSON is a lightweight data-interchange format.
+
+00:00:24.771 --> 00:00:29.633 
+It is easy for humans to read and write
+
+00:00:29.633 --> 00:00:34.676 
+and easy for machines to parse and generate.
```

Comment:
`parse` pronunciation is a little bit weird... it seems that he says "pause". But maybe it is just me.

Reply:
This seems to be fixed in video, resolving.

Reply:
mmm not really, it still feels weird to my ear... but maybe it is just me!

Reply:
I had another listen after the changes and it feels better? @handealptekin has done another iteration but I was not hearing the problem in the first place :see_no_evil: